e2e/binary_test.go

```diff
@@ -346,6 +343,9 @@ func TestSplitMergeBinary(t *testing.T) {
 	assertCommand(t, dockerApp, "split", "remerged", "-o", "splitted.dockerapp")
 	defer os.RemoveAll("splitted.dockerapp")
 	assertCommandOutput(t, "envvariables-inspect.golden", dockerApp, "inspect", "splitted")
+	// test inplace
+	assertCommand(t, dockerApp, "merge", "splitted")
+	assertCommand(t, dockerApp, "split", "splitted")
```

Comment:
nit: is `splitted` really grammatically correct?

Reply:
Neither is remerged probably :p

Reply:
Since it's a test, I don't think we should split hairs :)

The past tense of `split` is `split`. Thanks English.